### PR TITLE
Break out Request Children into their own component make Search UI similar to Requests UI

### PR DESCRIPTION
**Pull request by dhruvb14**
Title says it all, no functionality changes just code...

### CODE REVIEW

**Comment by dhruvb14** [comment]
Refactored Search to match Requests UI so you can choose episodes inline without leaving the search page, Temporarily removed infinite scroll on requests management screen since it has a bug which I was unable to resolve immediately...

File: src/Ombi/ClientApp/app/requests/tvrequests.component.ts [hunk 1]
```diff
@@ -227,13 +229,11 @@ export class TvRequestsComponent implements OnInit, OnDestroy {
         }
     }
     private removeChildRequestFromUi(key: IChildRequests) {
-        this.tvRequests.forEach((val) => {
```
Comment (tidusjar) [inline src/Ombi/ClientApp/app/requests/tvrequests.component.ts]: Do we no longer need this?

Comment (dhruvb14) [reply src/Ombi/ClientApp/app/requests/tvrequests.component.ts]: Checked in new update removing unused functions and updated type interface on a missed typed

**Comment by tidusjar** [comment]
Let me know when this is ready to merge on! :)

**Comment by dhruvb14** [comment]
It’s ready

**Comment by tidusjar** [comment]
Excellent!!

**Pull request merged**